app/helpers/finance_helper.rb

```diff
@@ -4,4 +4,12 @@ module FinanceHelper
   def number_to_pounds(number)
     number_to_currency number, precision: 2, unit: "£"
   end
+
+  def float_to_percentage(number)
+    number_to_percentage(number * 100, precision: 0)
+  end
+
+  def band_to_identifier(index)
+    ("A".ord + index).chr
+  end
```

Comment:
We are doing the same in the payment calculator and the output payment aggregator.

I wonder if an easier thing for us would be have something like `bands_with_identifiers` method on the contract model. Or even adding band name to the database. Thoughts?

Reply:
It's really only a decorator. Bands are ordered correctly defined by the limits, and I don't think they're needed apart for for display purposes. I did think about having them as a locale based rather than a fixed string. That may be better and allow for more flexibility when we start with D and potentially other bands based on the contractual changes that are happening. I see it more like the percentage and pound display, so having the other "output" displays using this as opposed to duplicating is probably a refactor that I'd prefer to do.